eng/pipelines/helix-platforms.yml

```diff
@@ -0,0 +1,311 @@
+  
+  # Linux musl ARM64
+  # Latest: Alpine 3.22
+  - name: helix_linux_musl_arm64_latest
+    value: (Alpine.322.Arm64.Open)Ubuntu.2204.ArmArch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.22-helix-arm64v8
+  
+  - name: helix_linux_musl_arm64_latest_internal
+    value: (Alpine.322.Arm64)Ubuntu.2204.ArmArch@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.22-helix-arm64v8
+  
+  # Oldest: Alpine 3.20
+  - name: helix_linux_musl_arm64_oldest
+    value: (Alpine.320.Arm64.Open)Ubuntu.2204.ArmArch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.20-helix-arm64v8
+  
+  - name: helix_linux_musl_arm64_oldest_internal
+    value: (Alpine.320.Arm64)Ubuntu.2204.ArmArch@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.20-helix-arm64v8
+  
+  # ===========================================
+  # Windows Platforms
+  # ===========================================
+  
+  # Windows x64
+  # Latest: Windows 11
+  - name: helix_windows_x64_latest
+    value: Windows.11.Amd64.Client.Open
+  
+  - name: helix_windows_x64_latest_internal
+    value: Windows.11.Amd64
+  
+  # Oldest: Windows 10
+  - name: helix_windows_x64_oldest
+    value: Windows.10.Amd64.Open
+  
+  - name: helix_windows_x64_oldest_internal
+    value: Windows.10.Amd64
+  
+  # Windows 11 with CET (Control-flow Enforcement Technology)
+  - name: helix_windows_x64_latest_cet
+    value: Windows.11.Amd64.Cet.Open
+  
+  # Windows Server
+  # Latest: Server 2025
+  - name: helix_windows_server_latest
+    value: Windows.Server2025.Amd64.Open
+  
+  # Oldest: Server 2019
+  - name: helix_windows_server_oldest
+    value: Windows.Server2019.Amd64.Open
+  
+  # Windows ARM64
+  # Latest: Windows 11
+  - name: helix_windows_arm64_latest
+    value: Windows.11.Arm64.Open
+  
+  - name: helix_windows_arm64_latest_internal
+    value: Windows.11.Arm64
+  
+  # Oldest: Windows 10
+  - name: helix_windows_arm64_oldest
+    value: Windows.10.Arm64.Open
+  
+  - name: helix_windows_arm64_oldest_internal
+    value: Windows.10.Arm64
+  
+  # ===========================================
+  # Browser/WASM Platforms
+  # ===========================================
+  
+  # Browser WASM
+  # Latest: Ubuntu 24.04
+  - name: helix_browser_wasm_latest
+    value: (Ubuntu.2404.Amd64)AzureLinux.3.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-24.04-helix-webassembly-amd64
+  
+  # Oldest: Ubuntu 22.04
+  - name: helix_browser_wasm_oldest
+    value: (Ubuntu.2204.Amd64)AzureLinux.3.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-22.04-helix-webassembly-amd64
+  
+  # ===========================================
+  # Android Platforms
+  # ===========================================
+  
+  # Android on Windows (ARM/ARM64)
+  # Latest
+  - name: helix_android_latest
+    value: Windows.11.Amd64.Android.Open
+  
+  # Oldest
+  - name: helix_android_oldest
+    value: Windows.10.Amd64.Android.Open
```

Comment:
@steveisok there's no oldest/latest for this queue, the windows version of the host PC doesn't correlate with the Android device versions. In fact the `Windows.10.Amd64.Android.Open` was deprecated and shouldn't be used anymore.